tests/testTriggers.go

```diff
@@ -652,6 +655,85 @@ func TriggerRestartVolDriver(contexts *[]*scheduler.Context, recordChan *chan *E
 	})
 }
 
+// TriggerRestartManyVolDriver restarts one or more volume drivers and validates app
+func TriggerRestartManyVolDriver(contexts *[]*scheduler.Context, recordChan *chan *EventRecord) {
+	defer ginkgo.GinkgoRecover()
+	event := &EventRecord{
+		Event: Event{
+			ID:   GenerateUUID(),
+			Type: RestartManyVolDriver,
+		},
+		Start:   time.Now().Format(time.RFC1123),
+		Outcome: []error{},
+	}
+
+	defer func() {
+		event.End = time.Now().Format(time.RFC1123)
+		*recordChan <- event
+	}()
+	driverNodesToRestart := getRandomNodes(Inst().NodesToGoDown)
+	chaosLevel := ChaosMap[RestartManyVolDriver]
+	var wg sync.WaitGroup
+	Step("get nodes bounce volume driver", func() {
+		for i := 0; i < chaosLevel; i++ {
+			for _, appNode := range driverNodesToRestart {
+				wg.Add(1)
+				go func(appNode node.Node) {
+					defer wg.Done()
+					Step(fmt.Sprintf("stop volume driver %s on node: %s", Inst().V.String(), appNode.Name), func() {
+						taskStep := fmt.Sprintf("stop volume driver on node: %s.",
+							appNode.MgmtIp)
+						event.Event.Type += "<br>" + taskStep
+						errorChan := make(chan error, errorChannelSize)
+						StopVolDriverAndWait([]node.Node{appNode}, &errorChan)
+						for err := range errorChan {
+							UpdateOutcome(event, err)
+						}
+					})
+				}(appNode)
+			}
+
+			Step("wait all the storage drivers to be stopped", func() {
+				wg.Wait()
```

Comment:
Should this wait be with timeout ? What if one of the nodes does not respond on stop vol driver ?